tests/unittests/test_check_access_token.py

```diff
@@ -0,0 +1,22 @@
+import unittest
```

Comment:
Unit tests for utils.py are missing.

Reply:
We have deleted `utils.py`. Only one function of this file was require which has been moved into `client.py`.